DifferentiationInterface/ext/DifferentiationInterfaceGTPSAExt/DifferentiationInterfaceGTPSAExt.jl

```diff
@@ -0,0 +1,13 @@
+module DifferentiationInterfaceGTPSAExt
+
+import DifferentiationInterface as DI
+using ADTypes: AutoGTPSA
+using GTPSA
+using LinearAlgebra
```

Comment:
Can we use explicit imports here too and add the same test as for other backends?